scripts/deleteElasticRecords.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from model import Work, ESWork
+from main import loadEnvFile
+from managers import DBManager, ElasticsearchManager
+
+
+def main():
+
+    '''Deleting ESC works that don't appear in the Postgresql database as works based on uuid'''
+
+    loadEnvFile('local-qa', fileString='config/{}.yaml')
+
+    dbManager = DBManager(
+        user= os.environ.get('POSTGRES_USER', None),
+        pswd= os.environ.get('POSTGRES_PSWD', None),
+        host= os.environ.get('POSTGRES_HOST', None),
+        port= os.environ.get('POSTGRES_PORT', None),
+        db= os.environ.get('POSTGRES_NAME', None)
+    )
+
+    esManager = ElasticsearchManager()
+    esManager.createElasticConnection()
+
+    dbManager.generateEngine()
+
+    dbManager.createSession()
+
+    batchSize = 1000
+    for work in esManager.session.query(ESWork) \
+        .yield_per(batchSize):       
```

Comment:
You'll need to construct a Search object the way that the API does, but there is an added complexity that ElasticSearch doesn't do "deep" paging well by default (not really it's intended use case).

The recommended solution is to use scan: see here: https://elasticsearch-dsl.readthedocs.io/en/latest/api.html#elasticsearch_dsl.Search.scan

The implementation should be straightforward, but I'm not sure about performance. If that's a problem there could be other approaches that we take.